test/test_traverse.py

```diff
@@ -70,3 +70,44 @@ def test_with_included_source_and_class_filter(self):
         ]
         self.assertEqual(filtered, ["Strong", "Strong"])
 
+    def test_with_depth_limit(self):
+        doc = Document("a **b** c **d**")
+        filtered = [t.node.__class__.__name__ for t in traverse(doc, depth=0)]
```

Comment:
It seems like all scenarios are covered -> OK.

Just for a little bit better readability, I would suggest to separate them by a new line and to add a short description on top of each - e.g. `# zero depth with root not included yields no nodes` for the 1st one (of course, we could put a message to the `assertEqual()` call as well, but I think it is not necessary in this case).

Reply:
Thanks for the feedback - there's a range of views on when and if to split into multiple test functions, and how to comment tests so I expected feedback in this area.

Separating the tests visually and adding a comment seems very pragmatic to me - I've made that change and pushed.

Reply:
Thank you. :)